etc/config/zig.amazon.properties

```diff
@@ -1,14 +1,17 @@
 compilers=&zig
-defaultCompiler=trunk
+defaultCompiler=z030
 
-group.zig.compilers=trunk
+group.zig.compilers=trunk:z030
```

Comment:
Can you add 0.2.0 to the list too (Even if a new version is out now)? I noticed it's being installed on the nodes, but it's not actually used here :)